lnbits/bolt11.py

```diff
@@ -244,19 +256,27 @@ def lnencode(addr, privkey):
 
 class LnAddr:
     def __init__(
-        self, paymenthash=None, amount=None, currency="bc", tags=None, date=None
+        self,
+        paymenthash=None,
+        amount=None,
+        currency="bc",
+        tags=None,
+        date=None,
+        fallback=None,
     ):
         self.date = int(time.time()) if not date else int(date)
         self.tags = [] if not tags else tags
         self.unknown_tags = []
         self.paymenthash = paymenthash
         self.signature = None
         self.pubkey = None
+        self.fallback = fallback
         self.currency = currency
         self.amount = amount
 
     def __str__(self):
-        pubkey = bytes.hex(self.pubkey.serialize()).decode()
+        assert self.pubkey
+        pubkey = bytes.hex(self.pubkey.serialize())
```

Comment:
- the `decode()` call disappeared. Is that on purpose?

Reply:
yes, i think this was an actual issue, introduced by switching from binascii package to builtin bytes.

Reply:
The whole class is unused... :)